scipy/stats/_distribution_infrastructure.py

```diff
@@ -1234,9 +1234,11 @@ def _log_real_standardize(x):
     return y.reshape(shape)[()]
 
 
-def _combine_docs(dist_family):
+def _combine_docs(dist_family, *, include_examples=True):
     fields = set(NumpyDocString.sections)
     fields.remove('index')
+    if not include_examples:
+        fields.remove('Examples')
```

Comment:
When generating docs (for `help`) for old distributions, we can't easily generate examples because the code for doing so doesn't know to get the parameter values from `_distparams.py`. That could be added as an enhancement later.